typescript/cli/src/registry/MergedRegistry.ts

```diff
@@ -92,7 +92,7 @@ export class MergedRegistry extends BaseRegistry implements IRegistry {
   }): Promise<void> {
     return this.multiRegistryWrite(
       async (registry) => await registry.addChain(chain),
-      `adding chain ${chain.chainName}`,
+      `Adding chain ${chain.chainName}`,
```

Comment:
The reason it was lower case is because it gets weaved into another log statement (see below). E.g. with this change you'll have 'Done Adding'

Reply:
Is it worth adding a little logic to update that first letter if it's appended or not?

Would just like to avoid the following logging:
```
✅ Balances are sufficient
All systems ready, captain! Beginning deployment...
Deploying ISM factory contracts
Deploying to alfajores from https://alfajores.celoscan.io/address/0x16F4898F47c085C41d7Cc6b1dc72B91EA617dcBb
Deploying to fuji from https://testnet.snowtrace.io/address/0x16F4898F47c085C41d7Cc6b1dc72B91EA617dcBb
skipping updating chain alfajores at github registry
updating chain alfajores at local registry at ./
done updating chain alfajores at local registry
skipping updating chain fuji at github registry
updating chain fuji at local registry at ./
done updating chain fuji at local registry
ISM factory contracts deployed
Deploying core contracts to alfajores, fuji
```

Reply:
this is a taste thing - but i think drive bys often work better as separate PRs because you might get comments that aren't related to the PR title or main body of work

Reply:
By all means feel free to improve this! I agree it's slightly off-putting but I wouldn't stress over it :)

Reply:
> this is a taste thing - but i think drive bys often work better as separate PRs because you might get comments that aren't related to the PR title or main body of work

agreed– this was a misuse of the "Drive-by changes" section for the PR. These were definitely drive-bys, but not solely related to the dry-run change. I think there is good use in keeping the Drive-by change section in our PR template, so long as those changes are tangentially related



> @nbayindirli By all means feel free to improve this! I agree it's slightly off-putting but I wouldn't stress over it :)

Will leave it be for now 👍